### PR TITLE
Update messages type to match current documentation

### PR DESCRIPTION
**Pull request by CoderBrad**
I have added the missing options and removed account since it is not listed in the docs.
https://github.com/paypal/paypal-js/issues/269

### CODE REVIEW

File: types/components/messages.d.ts [hunk 1]
```diff
@@ -13,7 +13,12 @@ export interface PayPalMessagesComponentOptions {
             size?: 10 | 11 | 12 | 13 | 14 | 15 | 16;
             align?: "left" | "center" | "right";
         };
+        ratio?: "1x1" | "1x4" | "8x1" | "20x1";
     };
+    placement?: "home" | "category" | "product" | "cart" | "payment";
+    onApply?: Function;
```
Comment (gregjopa) [inline types/components/messages.d.ts]: Let's avoid using `Function` and instead type these out as functions accept an object as the arg and return nothing.
```suggestion
    onApply?: (data: Record<string, unknown>) => void;
```

File: types/components/messages.d.ts [hunk 2]
```diff
@@ -13,7 +13,12 @@ export interface PayPalMessagesComponentOptions {
...
+    onClick?: Function;
```
Comment (gregjopa) [inline types/components/messages.d.ts]:
```suggestion
    onClick?: (data: Record<string, unknown>) => void;
```

File: types/components/messages.d.ts [hunk 3]
```diff
@@ -13,7 +13,12 @@ export interface PayPalMessagesComponentOptions {
...
+    onRender?: Function;
```
Comment (gregjopa) [inline types/components/messages.d.ts]:
```suggestion
    onRender?: (data: Record<string, unknown>) => void;
```

Comment (gregjopa) [reply types/components/messages.d.ts]: When testing locally I noticed that these callbacks do have data that's passed in. I don't think we need to type it all out at this point but it's good to call it out with `data: Record<string, unknown>`. Ex:
Screen Shot 2022-07-29 at 9 50 49 AM

Comment (CoderBrad) [reply types/components/messages.d.ts]: Yea, that was my delay in accepting. I was noticing paypal-messaging-components tests seems to have some data being passed in but tracing what and where it goes was getting a little confusing.

**Review by gregjopa — APPROVED** [review]

**Comment by gregjopa** [comment]
Thanks for your contribution @CoderBrad! 💯

**Pull request merged**